File: FrontEnd/src/app/models/invoice.ts
import {Invoice_Detail} from './invoice_detail'

export class Invoice{

    NameCli : string;
    
    IdCli : string;

    Total : number;
    
    Invoice_Detail : Invoice_Detail[];
    
    constructor(NameCli : string, IdCli : string,Total : number,Invoice_Detail : Invoice_Detail[]) {
        this.NameCli =NameCli;
    
        this.IdCli =IdCli;
    
        this.Total =Total;
        
        this.Invoice_Detail =Invoice_Detail;
    }

}

File: BackEnd/BackEnd/Domain/Models/Invoice.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace BackEnd.Domain.Models
{
    public class Invoice
    {
        public int Id { get; set; }

        [Required]
        [Column(TypeName = "varchar(50)")]
        public string NameCli { get; set; }
        [Required]
        [Column(TypeName = "varchar(50)")]
        public string IdCli { get; set; }

        public DateTime DateReg { get; set; }
        [Required]
        [Column(TypeName = "money")]
        public double Total { get; set; }


        public int UserId { get; set; }

        public Usersys User { get; set; }

        public List<Invoice_Detail> Invoice_Detail { get; set; }


    }
}
